main/sentry.py

```diff
@@ -146,3 +161,49 @@ def validate_config(cls):
                 )
             )
         )
+
+
+class SentryMonitorConfig:
+    """
+    Custom config for SentryMonitor
+    https://docs.sentry.io/product/crons/getting-started/http/#creating-or-updating-a-monitor-through-a-check-in-optional
+    """
+
+    MAX_RUNTIME_DEFAULT = 30  # Our default is 30 min
+    FAILURE_THRESHOLD_DEFAULT = 1
+    FAILURE_THRESHOLD = {
+        SentryMonitor.INDEX_AND_NOTIFY: 6,
```

Comment:
```suggestion
       # Index and notify runs every 5min, there is a posibility that it might fail, but we are okay for less then 6 fails
        SentryMonitor.INDEX_AND_NOTIFY: 6,
```